udata/core/reuse/views.py

```diff
@@ -97,11 +99,37 @@ def get_context(self):
         context.update(
             followers=followers,
             can_edit=ReuseEditPermission(self.reuse),
-            discussions=Discussion.objects(subject=self.reuse)
+            discussions=Discussion.objects(subject=self.reuse),
```

Comment:
No, keep the trailing coma.
When possible, we keep extra comas as Python allows it and it produces cleaner diffs when we add a line.